apps/frontend/src/components/cards/ComplianceChart.vue

```diff
@@ -45,7 +47,12 @@ export default class ComplianceChart extends Vue {
             show: true,
             value: {
               color: '#99a2ac',
-              fontSize: '2rem'
+              fontSize: '2rem',
+              formatter: (val: number) =>
+                `${(
+                  Math.trunc(Math.pow(10, MAX_DECIMAL_PRECISION) * val) /
```

Comment:
Mathematically speaking, both `Math.pow()` functions cancel each other out. So neither is really necessary and could be rewritten to just `Math.trucn(val).toFixed(MAX_DECIMAL_PRECISION)`, right?

Reply:
nope cause math.trunc cuts of all decimal values.  consequently i bring as much decimal precision as i need over to the left of the decimal point, chop it all off with trunc, and then shove it back over the dot again with the divide